### PR TITLE
Topics

### PR DESCRIPTION
**Pull request by catalin-ilea**

### CODE REVIEW

File: topics-transformer@.service [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=Topics transformer
+After=vulcan.service
+Requires=docker.service
+Wants=topics-transformer-sidekick@%i.service
+
+[Service]
+Environment="DOCKER_APP_VERSION=v1.0.0"
```
Comment (Zapadlo) [inline topics-transformer@.service]: Do you mean to hardcode?

Comment (catalin-ilea) [inline topics-transformer@.service]: Nope. Updated.

**Comment by tosan88** [comment]
:+1:

**Comment by BerniVarga** [comment]
+1

**Pull request merged**